src/components/Alert.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import MuiAlert from '@material-ui/lab/Alert';
+import AlertTitle from '@material-ui/lab/AlertTitle';
+import Text from './Text';
+
+function Core({
+  descriptionId = '',
+  description,
+  descriptionValues,
+  alertChildren,
+  children,
+  ...rest
+}) {
+  return (
+    <MuiAlert {...rest}>
+      {children}
+      {descriptionId ? (
+        <FormattedMessage
+          id={descriptionId}
+          values={descriptionValues}
+        />
+      ) : (
+        description
+      )}
+      {alertChildren}
+    </MuiAlert>
+  );
+}
+
+export default function CustomAlert({
+  titleId = '',
+  titleTxt,
+  titleValues,
+  children: alertChildren,
+  ...rest
+}) {
+  const isTitle = titleId || titleTxt;
+  return (
+    <Core {...rest} alertChildren={alertChildren}>
+      {isTitle ? (
+        <AlertTitle>
+          {titleId ? (
+            <Text id={titleId} values={titleValues} />
```

Comment:
For some reason I'm not seeing translated titleIds in the UI. For example:
![image](https://user-images.githubusercontent.com/9775178/125004518-8f1ae300-e00e-11eb-94ff-a9e0ea001aee.png)

Reply:
I don't know how I didn't notice this. Maybe my brain was like, "oh, there's a title that's populated. This works!". Good catch.